azurelinuxagent/common/cgroup.py

```diff
@@ -86,23 +86,28 @@ def _get_parameters(self, parameter_name, first_line_only=False):
         except IndexError:
             parameter_filename = self._get_cgroup_file(parameter_name)
             logger.error("File {0} is empty but should not be".format(parameter_filename))
-        except CGroupsException:
-            return None
+            raise CGroupsException("File {0} is empty but should not be".format(parameter_filename))
+        except CGroupsException as e:
+            raise e
```

Comment:
does this change the traceback in the exception?

maybe this instead?
>
       except CGroupsException:
            raise